engines/sci/engine/script_patches.cpp

```diff
@@ -3684,8 +3684,38 @@ static const uint16 phant1RatPatch[] = {
 	PATCH_END
 };
 
+// In Phantasmagoria the cursor’s hover state will not trigger on any of the buttons 
```

Comment:
Others have complained at me in the past when I sneaked some Unicode into the source code, so please make sure these beautiful, perfect, typographically correct apostrophes are turned into abominable ASCII typewriter apostrophes. :)